conf/cassandra.yaml

```diff
@@ -1575,29 +1575,24 @@ drop_compact_storage_enabled: false
 #    - 127.0.0.1
 #    - 127.0.0.0/31
 
-# Enables tracking warnings/aborts across all replicas for reporting back to client.
+# Enables read thresholds (warn/fail) across all replicas for reporting back to the client.
```

Comment:
We should update [the entry in `NEWS.txt`](https://github.com/apache/cassandra/blob/ab192b26a28e84f025d6be475786067f22f97cd5/NEWS.txt#L92-L101), it is still mentioning the previous properties.